client/src/styles/legacy/_style.scss

```diff
@@ -1354,13 +1354,17 @@ form.member-profile-form {
 
 .cms-panel {
   .cms-panel-toggle {
+    display: flex;
+    justify-content: flex-end;
+    padding: 0;
 
     .toggle-collapse,
     .toggle-expand {
-      height: 54px;
```

Comment:
As the parent `.cms-panel-toggle` is now always `display: flex`, these no longer need a hard-coded height as they’ll stretch automatically.